frontend/src/components/layout/navigation/whatsnew/WhatsNewMenu.tsx

```diff
@@ -84,6 +92,22 @@ export const WhatsNewMenu = (props: Props) => {
       });
     },
   });
+
+  const ctaUpgradePhones =
+    props.runtimeData &&
+    props.runtimeData !== undefined &&
+    !props.profile.has_phone &&
+    !props.profile.has_premium &&
+    isPremiumAvailableInCountry(props.runtimeData) ? (
+      <OutboundLink
+        to={getPhoneSubscribeLink(props.runtimeData)}
+        eventLabel={l10n.getString("whatsnew-feature-phone-upgrade-cta")}
+        className={styles.cta}
+      >
+        <span>{l10n.getString("whatsnew-feature-phone-upgrade-cta")}</span>
+      </OutboundLink>
+    ) : null;
```

Comment:
I refactored this such that this element is getting passed to `WhatsNewContent` from `WhatsNewEntry`

Reply:
OK, passing the whole component is even more generic, but works for me.